### DISCUSSION TITLE
Maintainer Wanted

### DISCUSSION THREAD

**Discussion by tecoholic** [tecoholic/ner-annotator#68] [Announcements]
As I haven't been working on any Machine Learning related projects for almost 2 years now, I am no longer invested enough to maintain the NER Tagger project anymore. There have been some quality improvements being made via pull requests by developers which I have not had the time to review, test and merge.

So, I would be happy to pass on the maintainer role to someone who uses this in their day to day life and want to see the improvements being made to this tool.

If you are interested, kindly drop a line below and I will add necessary repository rights to you.

**Comment by tecoholic** [comment]
Pinging everyone who has made a contribution to the project so far: @alvi-khan, @leonkunert, @LelViLamp, @Totalus, @lyihaoo @Faran-Javaid.

**Comment by alvi-khan** [comment]
I can take on this role if you want @tecoholic, but I haven't really moderated open-source projects before so it would be great if you could answer any questions I might have from time to time.

**Reply by tecoholic** [reply]
@alvi-khan Than you for stepping in. This is a fairly small project with limited contributions. If you are using the software regularly, it should be fairly easy to manage.

**Reply by tecoholic** [reply]
@alvi-khan I have invited you to be the collaborator on the project. Your should be able to push to the main branch and make releases. For some reason the github actions pushing the release is broken. But, I think we should be able to download the artifacts and upload it to a manual release. I have always built things manually and uploaded the binaries anyway. I setup the github action to automate it for me, but never made a full release using it. Kindly let me know if you don't have access to anything, I will find the setting and correct it.

**Reply by alvi-khan** [reply]
@tecoholic Unfortunately it seems that I can't approve the pull requests I created. I'm unsure if this is an issue with the settings or if this is just how pull requests work, but please take a look into it when you have the time.

**Reply by tecoholic** [reply]
@alvi-khan I tried finding a setting to get this working. Couldn't find the right one. So, I have added a proxy approval in place of you. I think you should be able to merge the PRs now.

**Reply by alvi-khan** [reply]
@tecoholic It also seems that main is a protected branch that I cannot push to. I'm unsure if non-admins can do that, but just so you know this means that I will have to create pull requests for any changes I make and you will in turn have to add proxy approvals for each pull request.

**Reply by tecoholic** [reply]
@alvi-khan I have removed the branch protections on the "main" branch. You should be able to make changes and merge pull requests. I hope this makes things smoother for you.

**Comment by tecoholic** [comment]
@alvi-khan Congratulations on the new release 🍾 🚀

**Reply by alvi-khan** [reply]
Thank you very much! I'm grateful for the opportunity to work with you on an application that has been so useful to me.

**Reply by tecoholic** [reply]
I should be the one to thank you for getting the project back to life :)
